Develop

orest-serafym opened the pull request.

MariKush committed:
added login and start testing boards

MariKush committed:
added a test to create a board

orest-serafym committed:
Filters flow test

MariKush committed:
add Delete board test

orest-serafym committed:
Update test

orest-serafym committed:
Delete example_test.js

orest-serafym committed:
Update file name

orest-serafym committed:
Delete data.json

orest-serafym committed:
Update

orest-serafym committed:
Merge pull request #1 from MariKush/test_boards

Test boards

MariKush committed:
Merge branch 'develop' into filters

MariKush committed:
Merge pull request #2 from MariKush/filters

Filters

karunandrii reviewed (changes requested):
1. isNewlyCreatedBoardDisplayed - Please, make proper assertion. Get all items, find by unique name and assert.
2. 'a=' + data.boardName + '' - use template literals to create strings
3. browser.maximizeWindow(); - can go to before hook
4. const assert = require('assert') - use chai instead
5. userLogin should be implemented in separate module
6. if you pass timeout into wait() functions - make sure system respond same way JUST FYI

orest-serafym committed:
Update tests

orest-serafym commented:
> 1. isNewlyCreatedBoardDisplayed - Please, make proper assertion. Get all items, find by unique name and assert.
> 2. 'a=' + data.boardName + '' - use template literals to create strings
> 3. browser.maximizeWindow(); - can go to before hook
> 4. const assert = require('assert') - use chai instead
> 5. userLogin should be implemented in separate module
> 6. if you pass timeout into wait() functions - make sure system respond same way JUST FYI



> 1. Not fixed cause it not my module (And there is not enough time)
> 2. fixed 
> 3. fixed
> 4. const assert = require('assert') - use chai instead (not a priority and no time)
> 5. fixed

karunandrii commented:
**const assert = require('assert') - use chai instead (not a priority and no time**) - chai is globally set, all you needed is change assertion to chai syntax.
